dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
-black
+black==19.10b0
 pytest
```

Comment:
?

Reply:
I use pipenv to install from the dev-requirements.txt file, and because black is still in "pre-release" it will refuse to install it. I can force pipenv by being specific about the version that I want. I can revert this change if you want